src/build_backend.py

```diff
@@ -70,10 +71,17 @@ def build_wheel(wheel_directory: str,
             '[console_scripts]',
             'cockpit-bridge = cockpit.bridge:main',
             'cockpit-askpass = cockpit._vendor.ferny.interaction_client:main',
+            'cockpit-ssh = cockpit.ssh:main',
         ],
     }
 
     with zipfile.ZipFile(f'{wheel_directory}/{wheel_filename}', 'w') as wheel:
+        def beipack_self(main: str) -> bytes:
+            from cockpit._vendor.bei import beipack
```

Comment:
This feels like a weird naming choice in beipack. The project is called "beipack", presumably it'll have that name on pypi, so the module name should be called that way, too? Also, doesn't this collide with https://github.com/allisonkarlitskaya/beipack eventually? Or do you want to merge these projects?

Reply:
The project contains beiboot and beipack, and I consider beiboot to be the primary "thing" and beipack a bit more like an enablement technology, although in some sense, they are peers.  They used to be called `bei.pack` and `bei.boot`.

The name `bei` was disallowed by PyPI as being potentially spammy, and `beiboot` is already taken by something unrelated, which left `beipack` as my third choice.  I'd have preferred to call it something else.